packages/design-picker/src/utils/designs.ts

```diff
@@ -11,12 +11,12 @@ export const getDesignPreviewUrl = (
 	if ( [ 'hannah', 'riley', 'gilbert' ].indexOf( slug ) >= 0 ) {
 		return `https://${ slug }starter.wordpress.com`;
 	}
-
 	let url = addQueryArgs( 'https://public-api.wordpress.com/wpcom/v2/block-previews/site', {
 		stylesheet: recipe?.stylesheet,
 		pattern_ids: recipe?.patternIds?.join( ',' ),
 		language: options.language,
 		viewport_height: 700,
+		source_site: 'patternboilerplates.wordpress.com',
```

Comment:
We need to update the tests in https://github.com/Automattic/wp-calypso/blob/trunk/packages/design-picker/src/utils/__tests__/designs.test.ts#L5.

Reply:
I updated it to only add `source_site` if `patternIds` are also sent, and updated tests